interface-spec.md

```diff
@@ -38,9 +43,7 @@ TODO: to what extent should we use typed signatures (`.equals(Node other)`) vers
 
 **Properties:**
 
-- `String .value` blank node name as a string (example: `_:blank3`)
-
-TODO: Does the value always start with an underscore?
+- `String .value` blank node name as a string without leading `_:` (example: `blank3`)
```

Comment:
> presumption that they start with _: is only based on the fact that nearly all concrete syntaxes use this notation. While unlikely, it's possible that another syntax would not use that form.

Which is why `_:` should  not be there, because it is syntax-specific?